lib/init.php

```diff
@@ -49,18 +54,20 @@ function beans_define_constants() {
 	define( 'BEANS_ADMIN_URL', BEANS_URL . 'admin/' );
 	define( 'BEANS_ADMIN_ASSETS_URL', BEANS_ADMIN_URL . 'assets/' );
 	define( 'BEANS_ADMIN_JS_URL', BEANS_ADMIN_ASSETS_URL . 'js/' );
-
 }
 
 add_action( 'beans_init', 'beans_load_dependencies', -1 );
 /**
  * Load dependencies.
  *
+ * @since 1.0.0
  * @ignore
+ * @access private
```

Comment:
The access is not private.  Go ahead and remove the `@access` tag.  We do want to ignore it though for the automatic doc generator.